test/compound_assignment_test.js

```diff
@@ -29,6 +29,33 @@ describe('compound assignment', () => {
     check(`a ^= 1`, `a ^= 1;`);
   });
 
+  it('handles compound modulo assignment', () => {
+    check(`
+      a %%= b
+    `, `
+      let a = __mod__(a, b);
```

Comment:
Good point, I didn't think of that. Hopefully any reasonable code wouldn't rely on that behavior, but yeah, would be good to address eventually.